features/guidance/collapse-ferry.feature

```diff
@@ -0,0 +1,90 @@
+@routing  @guidance @collapsing
+Feature: Collapse
+
+    Background:
+        Given the profile "car"
+        Given a grid size of 20 meters
+
+    Scenario: Collapse Steps While On Ferry
+        Given the node map
+            """
+            j    a   c   b    k
+
+
+                        d
+
+
+                                    f
+                                e
+            """
+
+        And the ways
+            | nodes | highway | route | name |
+            | jacbk | primary |       | land |
+            | ad    |         | ferry | sea  |
+            | bd    |         | ferry | sea  |
+            | cd    |         | ferry | sea  |
+            | de    |         | ferry | sea  |
+            | ef    | primary |       | land |
+
+        When I route I should get
+            | waypoints | route              | turns                                      | modes                         |
+            | f,j       | land,sea,land,land | depart,notification right,turn left,arrive | driving,ferry,driving,driving |
+
+    Scenario: Collapse Steps While On Ferry
```

Comment:
I believe we could remove this test. It looks like the only difference here to the test above is an additional coordinate on the ferry route, which should not influence any turn decisions.